js/README.md

```diff
@@ -26,4 +26,24 @@ from { HDWallet } from 'trust-wallet-core';
 
 HDWallet.create(128);
 //=> "army cook device assault vendor case coast rose dismiss lady dust point!"
-```
\ No newline at end of file
+```
+
+## Documentation
+
+### Platforms
+
+`trust-wallet-core` only supports `macOS/OSX` at the moment. `Linux` and `Windows` support is forthcoming.
```

Comment:
```suggestion
`trust-wallet-core` only supports `macOS/OSX` at the moment. `Linux` and `Windows` support are forthcoming.
```

Reply:
The use of `is` is correct here since the `support` (singular) is what is forthcoming.

Reply:
👍